jovian/utils/api.py

```diff
@@ -154,3 +154,30 @@ def commit_records(gist_slug, tracking_slugs, version=None):
         return res.json()['data']
     else:
         raise ApiError('Data logging failed: ' + _pretty(res))
+
+
+def notify_on_slack(data):
+    """Push data to Slack, if slack is integrated with jovian account"""
+    url = _u('/slack/notify')
+    res = post(url, json=data, headers=_h())
+    if res.status_code == 200:
+        return res.json()
+    else:
+        raise ApiError('Slack trigger failed: ' + _pretty(res))
+
+
+def add_slack():
```

Comment:
as mentioned above, this needs to be moved to a separate file `jovian.utils.slack`, for python 2.7 support. Also, test out the CLI on python 2.7 to make sure it works properly. You can create a conda environment: `conda create -n slacktest python=2.7`